data/ga/legislature/Deborah-Silcox-b423e42d-eead-4f58-b82a-7c76a1a311b4.yml

```diff
@@ -0,0 +1,38 @@
+id: ocd-person/b423e42d-eead-4f58-b82a-7c76a1a311b4
+name: Deborah Silcox
+given_name: Deborah
+family_name: Silcox
+email: deborah.silcox@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/silcox-deborah-4887.jpg
+party:
+- name: Republican
+roles:
+- type: lower
+  jurisdiction: ocd-jurisdiction/country:us/state:ga/government
```

Comment:
@sroomf this person served in GA House district 52 from Jan 2019 until Jan 2021.